server/package.json

```diff
@@ -75,6 +75,8 @@
 	"devDependencies": {
 		"chai": "4.2.0",
 		"eslint-plugin-unused-imports": "^2.0.0",
+		"husky": "^4.3.8",
```

Comment:
@LVMBDV remove ^ . We always set the version in a static way to avoid any random package issues in future